dac_calibration_server.py

```diff
@@ -102,15 +99,11 @@ def loadServerSettings(self):
             'bandwidthIQ': 0.4, #original default: 0.4
             'bandwidthZ': 0.13, #original default: 0.13
             'maxfreqZ': 0.45, #optimal parameter: 10% below Nyquist frequency of dac, 0.45
-            'maxvalueZ': 5.0, #optimal parameter: 5.0, from the jitter in 1/H fourier amplitudes
-            'dither': True #enable dithering on Z.
+            'maxvalueZ': 5.0 #optimal parameter: 5.0, from the jitter in 1/H fourier amplitudes
         }
         for key in keys.SERVERSETTINGVALUES:
             default = defaults.get(key, None)
-            keyval = yield reg.get(key, False, default)
-            if not isinstance(keyval, bool):
-                #keyval is a number, in labrad units
-                keyval = keyval
+            keyval = default
             print key, ':', keyval
             d[key] = keyval
```

Comment:
Yes, it's bogus. I didn't refactor it yet because I am trying to keep the commits very small to facilitate reviews.